datahub-web-react/src/graphql/assertion.graphql

```diff
@@ -0,0 +1,88 @@
+fragment assertionDetails on Assertion {
+    urn
+    type
+    platform {
+        urn
+        properties {
```

Comment:
curious. why do we need both? is this for backward compatibility?

Reply:
both meaning properties and info

Reply:
yeah it was mainly compatibility - we haven't done the full audit on the UI for removing info :(